### PR TITLE
XW-1581 | fixes due to Design System footer changes

### PR DESCRIPTION
**Pull request by Warkot**
@Wikia/x-wing 

I'll look if there are more changes needed. Anyway, we have to merge it do `master`, not earlier and not later than when realising the footer itself.

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/mercury/old/ArticlePageObject.java [hunk 1]
```diff
@@ -22,7 +22,9 @@
   private List<WebElement> topContributorsThumbs;
   @FindBy(css = "figure.article-image a")
   private List<WebElement> singleImgLink;
-  @FindBy(css = "svg.logo")
+  @FindBy(css = "footer.global-footer")
+  private WebElement footer;
+  @FindBy(css = "svg.global-footer__fandom-logo")
```
Comment (rogatty) [inline src/test/java/com/wikia/webdriver/elements/mercury/old/ArticlePageObject.java]: This won't be visible on non-english wikis.

Comment (Warkot) [inline src/test/java/com/wikia/webdriver/elements/mercury/old/ArticlePageObject.java]: Yes.

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by Warkot** [comment]
Something went crazy during merging `origin/master`, so I recreated the PR here https://github.com/Wikia/selenium-tests/pull/1395.

**Pull request closed without merging**